dev/com.ibm.ws.cdi.internal/src/com/ibm/ws/cdi/internal/interfaces/ExtensionArchiveProvider.java

```diff
@@ -29,6 +38,11 @@ public interface ExtensionArchiveProvider {
      * @return a collection of additional extension archives to add
      * @throws CDIException if an unexpected exception is encountered
      */
-    Collection<ExtensionArchive> getArchives(CDIRuntime cdiRuntime, WebSphereCDIDeployment deployment) throws CDIException;
+    default Collection<ExtensionArchive> getDiscoveredArchives(CDIRuntime cdiRuntime, WebSphereCDIDeployment deployment) throws CDIException {
+        return Collections.emptySet();
+    }
```

Comment:
This name change doesn't make sense. The archives added by implementors of this interface are not "discovered" - they're created anew.

Reply:
I was thinking of the fact we LiteExtensionProvider was creating archives for BCEs discovered through the classloader. I have no objection to changing the name though.